lib/readiness.js

```diff
@@ -47,14 +47,25 @@ bedrock.events.on('bedrock-server.readinessCheck', async () => {
   await pWaitFor(async () => {
     logger.debug('Running readiness check...');
     let ready = false;
+    let dependencies;
     try {
-      ({ready} = await check());
+      ({ready, dependencies} = await check());
     } catch(error) {
       logger.error('An error occurred during the readiness check.', {error});
       return false;
     }
     if(!ready) {
-      logger.debug('The readiness check failed. Retrying...');
+      let names = '';
+      for(const name in dependencies) {
+        if(dependencies[name].ready === false) {
+          if(names === '') {
+            names = name;
+          } else {
+            names = `${names}, ${name}`;
+          }
+        }
+      }
```

Comment:
This is a cleaner way to filter and produce the dependency names in a comma-delimited list:

```suggestion
      const names = Object.keys(dependencies)
        .filter(n => !dependencies[n].ready)
        .join(', ');
```

That being said -- do we want a comma-delimited list or do we want to include the array of names in the logging data? We could also do both by not joining until we create the message:

```
logger.debug(`The readiness for "${names.join(', ')}" failed. Retrying...`, {dependencies: names});
```

Reply:
Updated